package.json

```diff
@@ -35,5 +35,22 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
```

Comment:
- [ ] It would be better if you add `react-redux` to the dependencies, `react-redux` will be needed to use Redux in the next projects
😄 💯

Reply:
resolved this